lib/ddtrace/span.rb

```diff
@@ -60,26 +58,22 @@ class Span
 
     attr_accessor :name, :service, :span_type,
                   :span_id, :trace_id, :parent_id,
-                  :status, :sampled,
-                  :tracer, :context
```

Comment:
![image](https://user-images.githubusercontent.com/583503/132775455-52ee22dc-3d31-42c0-b62e-f1c222e1dbdc.png)

Reply:
Haha, trust the tests!